lib/active_merchant/billing/gateways/plexo.rb

```diff
@@ -264,7 +269,9 @@ def parse(body)
         JSON.parse(body)
       end
 
-      def build_url(action, base)
+      def build_url(action, base, options, parameters)
+        return "#{base}/#{parameters[:payment_id]}" if action == 'status' && parameters[:payment_id]
```

Comment:
💬 Comment:

Maybe a more "holistic" approach could be taken here to add clarity to the `build_url` method and maybe bring the entire url logic here by moving line 295 to here.